### ISSUE TITLE
About pretrained weights and results

### ISSUE THREAD

**Issue by DaiMaocai** [Dreemurr-T/BAID#9]
Hello author, I have some questions about weights and reproducing results：
1. About pre-trained weights：
Your code and the pre-trained weights provided are all epoch_99.pth, so I would like to ask why you use this weight instead of others. I don't seem to see the relevant code.

![Image](https://github.com/user-attachments/assets/6ed878f9-8ec4-4d7d-94ff-b7e87174c60f)

2.About Training：
I also used the epoch_99.pth mentioned in your code by default, but this pre-trained weight was trained by myself (not sure it is the best), and I did not use the weight you provided in Drive for training. After the training was completed, I chose the weight with the lowest Validation loss as the weight for testing.
there is testing result：

![Image](https://github.com/user-attachments/assets/58ac7c09-a506-414e-a40e-01cf9261fb25)
Both SRCC and PCC results are much lower than your results.

3.I also tested other training weights, even lower, so I want to know why my results are so different. According to my understanding, the results of the reproduction should not be much different from yours. Is it because the pre-training weights used at the beginning were not selected correctly, or is there any other reason? If you know, please let me know, thank you very much 
@Dreemurr-T @Dreemurr-T @Dreemurr-T
